.changes/v2.23.0/645-features.md

```diff
@@ -0,0 +1,19 @@
+* Added a new type `CseKubernetesCluster` to manage Container Service Extension Kubernetes clusters [GH-645]
```

Comment:
```suggestion
* Added type `CseKubernetesCluster` to manage Container Service Extension Kubernetes clusters [GH-645]
```
I think we can remove "new " or "a new" from all the statements in this entry

Reply:
Fixed